.fixtures.yml

```diff
@@ -2,7 +2,8 @@ fixtures:
   forge_modules:
     stdlib: puppetlabs/stdlib
     apt: puppetlabs/apt
-    zypprepo: darin/zypprepo
+    zypprepo: puppet/zypprepo
```

Comment:
The `zypprepo` module has been adopted by Vox Populi and made compatible with Puppet 6. :1st_place_medal: